packages/@internationalized/date/src/index.ts

```diff
@@ -38,7 +38,17 @@ export {IslamicCivilCalendar, IslamicTabularCalendar, IslamicUmalquraCalendar} f
 export {HebrewCalendar} from './calendars/HebrewCalendar';
 export {EthiopicCalendar, EthiopicAmeteAlemCalendar, CopticCalendar} from './calendars/EthiopicCalendar';
 export {createCalendar} from './createCalendar';
-export {toCalendarDate, toCalendarDateTime, toTime, toCalendar, toZoned, toTimeZone, toLocalTimeZone} from './conversion';
+export {
+  toCalendarDate,
+  toCalendarDateTime,
+  toTime,
+  toCalendar,
+  toZoned,
+  toTimeZone,
+  toLocalTimeZone,
+  fromDate,
+  fromAbsolute as fromEpoch
```

Comment:
I think we'd like to keep this named `fromAbsolute`. We used the term absolute for `parseAbsolute` as well. They are both representations of the same thing, either in string or number form. In the docs we can explain that the number uses the Unix epoch (1970).

Reply:
Hey @devongovett 
Totally make sense, I should not have changed this common term.
I have updated it back to `fromAbsolute` and updated the respective docs as well.

please review it further,

Thanks